docs/pages/bare/hello-world.mdx

```diff
@@ -7,7 +7,7 @@ description: An overview of bare React Native app with Expo tools.
 import { BoxLink } from '~/ui/components/BoxLink';
 import { Terminal } from '~/ui/components/Snippet';
 
-A bare React Native app is a project where developers make direct changes to their native `ios` and `android` project directories rather than continuously generating them on demand using the [Expo config (**app.json**) and prebuild](/workflow/prebuild). All tools and services offered by Expo including [EAS](/eas), Expo CLI, and the libraries in the Expo SDK, are built around bare React Native apps.
+A bare React Native app is a project where developers make direct changes to their native **android and **ios** project directories rather than continuously generating them on demand using the [Expo config (**app.json\*\*) and prebuild](/workflow/prebuild). All tools and services offered by Expo including [EAS](/eas), Expo CLI, and the libraries in the Expo SDK, are built around bare React Native apps.
```

Comment:
```suggestion
A bare React Native app is a project where developers make direct changes to their native **android and **ios** project directories rather than continuously generating them on demand using the [Expo config (**app.json**) and prebuild](/workflow/prebuild). All tools and services offered by Expo including [EAS](/eas), Expo CLI, and the libraries in the Expo SDK, are built around bare React Native apps.
```

Looks like Prettier side-effect.

Reply:
Nice catch! It seems so.